### ISSUE TITLE
widget datetime

### ISSUE THREAD

**Issue by acmorganti** [mjbeller/web2py-starter#4]
Hello,

I am just creating a single SQLFORM from a model and there is a datetime field. The datetime widget, that is open a calendar, is not running.

I tried to find the problem but no success.

If I understood correctly, the all funcionalities from web2py are presented in this scaffold, is'n it?

Do you have any idea?

Thanks
BR
André

**Comment by mjbeller** [comment]
Yes - all the web2py functions are included in the scaffold but I also override the default datetime widget with a more flexible open source datetime widget library.  That's included in the repo but it may not be loading.

**Comment by acmorganti** [comment]
Hi,
Thanks your time.
I found in static/plugins datepicker and timepicker. I added the lines:
$(function () {
          $('#table_datefield').datepicker();
    })  ;

    $(function () {
      $('#table_timefield').timepicker();
    });

For date, perfect, it's running. Very nice!
For time, I am having the error in console

> VM876:1 Uncaught TypeError: $(...).timepicker is not a function
>     at <anonymous>:1:29
> (anonymous) @ VM876:1

I tried to import timepicker.js in create.html but this error persist.
Do you have an idea how to use timepicker plugin?

Sorry to bother you with this topic.
Best regards
André

**Comment by mjbeller** [comment]
I haven't played with timepicker, let me know what you find.
